src/api/deploy-test-suite/helpers/generate-test-run-message.js

```diff
@@ -17,7 +17,7 @@ const testRunMessageValidation = Joi.object({
   cluster_name: Joi.string().equal('ecs-public'),
   name: repositoryNameValidation,
   image: repositoryNameValidation,
-  image_version: Joi.number().equal('latest'),
+  image_version: Joi.string().pattern(/\d+\.\d+\.\d+/),
```

Comment:
we have different `tagValidation` lower down? Should we just put them both in common and use everywhere?

Reply:
thinking about it we might as well just use the existing version validation